alembic/versions/b99155654de_sunset_all_future_metrofirefox.py

```diff
@@ -0,0 +1,27 @@
+"""sunset all future MetroFirefox
+
+Revision ID: b99155654de
+Revises: 56f5cdf9bcdb
+Create Date: 2015-07-31 15:51:26.282796
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'b99155654de'
+down_revision = '56f5cdf9bcdb'
+
+from alembic import op
+
+
+def upgrade():
+    op.execute("""
+        UPDATE product_versions
+        SET sunset_date = NOW()
+        WHERE
+            product_name = 'MetroFirefox' AND
+            sunset_date >= NOW()
+    """)
+
+
+def downgrade():
```

Comment:
I don't think it's worth trying to undo the product_versions update in the case of a downgrade, but you should load the stored proc again (the assumption being that if we downgrade, we want to revert the stored proc change), just `load_stored_proc(op, ['update_product_versions.sql'])` again